egs/lre07/v2/local/dnn/run_nnet2_multisplice.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+# This script is based on run_nnet2_multisplice.sh in
+# egs/fisher_english/s5/local/online. It has been modified
+# for speaker recognition.
+
+. cmd.sh
+
+
+stage=1
+train_stage=-10
+use_gpu=true
+set -e
+. cmd.sh
+. ./path.sh
+. ./utils/parse_options.sh
+
+
+# assume use_gpu=true since it would be way too slow otherwise.
+
+if ! cuda-compiled; then
+  cat <<EOF && exit 1 
+This script is intended to be used with GPUs but you have not compiled Kaldi with CUDA 
+If you want to use GPUs (and have them), go to src/, and configure and make on a machine
+where "nvcc" is installed.
+EOF
+fi
+parallel_opts="-l gpu=1" 
+num_threads=1
+minibatch_size=512
+dir=exp/nnet2_online/nnet_ms_a
+mkdir -p exp/nnet2_online
+
+
+# Stages 1 through 5 are done in run_nnet2_common.sh,
+# so it can be shared with other similar scripts.
+local/dnn/run_nnet2_common.sh --stage $stage
+
+if [ $stage -le 6 ]; then
+  if [[ $(hostname -f) == *.clsp.jhu.edu ]]; then
+    utils/create_split_dir.pl /export/b0{6,7,8,9}/$(USER)/kaldi-dsata/egs/fisher_english/s5/$dir/egs/storage $dir/egs/storage
```

Comment:
Can you change the location the egs are dumped to
`/export/b0{6,7,8,9}/$(USER)/kaldi-data/egs/lre07/v2/$dir/egs/storage`